fluentd/configs.d/filter-post-z-retag-two.conf

```diff
@@ -4,12 +4,16 @@
   @label @OUTPUT
 </match>
 
-<match journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_**>
+# Remove audit logs where type != virt-control
+@include filter-exclude-audit.conf
+
+<match journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_** audit.log**>
   @type rewrite_tag_filter
   @label @OUTPUT
   rewriterule1 message .+ output_ops_tag
   rewriterule2 MESSAGE .+ output_ops_tag
-  rewriterule3 log .+ output_tag_tag
+  rewriterule3 log .+ output_ops_tag
```

Comment:
We are removing this? or is that in a separate PR?

Reply:
This should be in a separate pr

Reply:
@nhosoi has already submitted the PR https://github.com/openshift/origin-aggregated-logging/pull/715